controllers/constant/webhook.go

```diff
@@ -1633,6 +1633,7 @@ spec:
                   type: object
                 type: array
             type: object
+            x-kubernetes-preserve-unknown-fields: true
```

Comment:
From the template
```
spec:
            description: |+
              * documentation of parameters available, but not validated by OpenAPI / CRD schema
            type: object
            x-kubernetes-preserve-unknown-fields: true
```

I think the clause should be aligned with line 1637 `type: object`